code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -1085,20 +1079,35 @@ pub mod pallet {
 
 		pub(crate) fn split_lock(
 			asset_id: T::AssetId,
-			existing_account_amount: T::Balance,
+			old_fnft_asset_account: &T::AccountId,
+			new_fnft_asset_account: &T::AccountId,
+			old_account_amount: T::Balance,
 			new_account_amount: T::Balance,
-			existing_asset_account: &T::AccountId,
-			new_asset_account: &T::AccountId,
 		) -> DispatchResult {
```

Comment:
I like the new `split_lock` method, which cleans up the calling method quite a bit. However, I think we should use the terms "existing" and "new" as using "old" and "new" infers that the old is being wholly replaced, which is not happening.

Reply:
agreed - updated in https://github.com/ComposableFi/composable/commit/c9c49555f0eec267ead832ab544f9acd662a0534